server/src/main/java/org/elasticsearch/cluster/SnapshotsInProgress.java

```diff
@@ -365,11 +401,7 @@ private boolean assertConsistent() {
         public ShardSnapshotStatus(StreamInput in) throws IOException {
             nodeId = in.readOptionalString();
             state = ShardState.fromValue(in.readByte());
-            if (SnapshotsService.useShardGenerations(in.getVersion())) {
-                generation = in.readOptionalString();
-            } else {
-                generation = null;
-            }
+            generation = in.readOptionalString();
```

Comment:
We can remove this in `master` because we're checking against version `7.6` in that method.